### ISSUE TITLE
[JENKINS-31634] Workflow editor font size too small

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/workflow-cps-plugin#1147]
<p>The font size of the workflow editor is very tiny. It seems to be smaller then all the other editor boxes in Jenkins.</p>

<p>Personally I think it should be the same font size then any other editor box, at least this should be the default.</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=cruftex">cruftex</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-31634" target="_blank">Workflow editor font size too small</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/tfennelly.png" title="tfennelly's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=tfennelly">tfennelly</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Minor
<li><b>component(s)</b>: workflow-cps-plugin
<li><b>label(s)</b>: accessibility, regression, usability, user-experience
<li><b>resolution</b>: Postponed
<li><b>resolved</b>: 2017-12-04T00:19:46+00:00
<li><b>votes</b>: 1
<li><b>watchers</b>: 3
<li><b>imported</b>: 2025-12-07
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>The font size of the workflow editor is very tiny. It seems to be smaller then all the other editor boxes in Jenkins.</p>

<p>Personally I think it should be the same font size then any other editor box, at least this should be the default.</p></pre>
</details>
</details>
